### PR TITLE
Data types overview

### PR DESCRIPTION
**Pull request by ZoyaNaumenko**

### CODE REVIEW

File: Exercises/data-types-overview/data-types-overview.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+//1. Declare a variable on each type using two options: plain creation, creation with constructor.
+let numberVariable = 123;
```
Comment (lizarusi) [inline Exercises/data-types-overview/data-types-overview.js]: always use `const` when variable not going to be change

File: Exercises/data-types-overview/data-types-overview.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+console.log(piu);
+//2. Iterate through it and collect amount of each data type to object
+let objOutput = {};
+let i = 0;
+
+for (i ; i < piu.length ; i++) {
+  let key = typeof piu[i];
+  if (objOutput[key] !== undefined ) {
+    objOutput[key] += 1;
```
Comment (lizarusi) [inline Exercises/data-types-overview/data-types-overview.js]: you can simplify condition

File: Exercises/data-types-overview/data-types-overview.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+//# Additional
+//1. Declare an array containing different data types.
+let piu = [213, 'test', 333, 675, {}, true];
+
+console.log(piu);
+//2. Iterate through it and collect amount of each data type to object
+let objOutput = {};
+let i = 0;
+
+for (i ; i < piu.length ; i++) {
```
Comment (lizarusi) [inline Exercises/data-types-overview/data-types-overview.js]: also try to use https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce here

**Review by lizarusi — CHANGES_REQUESTED** [review]

**Review by Mike-Savin — APPROVED** [review]
👍

**Pull request merged**